### PR TITLE
Prepare to cut 0.17.0.pre1

### PR DESCRIPTION
**Pull request by digitalextremist**

### CODE REVIEW

**Comment by digitalextremist** [comment]
@tarcieri do you mind showing me how to manage coveralls? The spam here is wow.

**Comment by tarcieri** [comment]
I just disabled it. Let me know if it's spamming any other projects

**Comment by digitalextremist** [comment]
@niamster it seems `jruby` is sending back objects named in a way the tests cannot understand.

**Comment by niamster** [comment]
all green

File: lib/celluloid/zmq/mailbox.rb [hunk 1]
```diff
@@ -1,7 +1,7 @@
 module Celluloid
   module ZMQ
     # Replacement mailbox for Celluloid::ZMQ actors
-    class Mailbox < Celluloid::EventedMailbox
+    class Mailbox < Celluloid::Mailbox::Evented
```
Comment (digitalextremist) [inline lib/celluloid/zmq/mailbox.rb]: @niamster this is a `0.17.0` change

File: Gemfile [hunk 2]
```diff
@@ -1,7 +1,14 @@
 source 'http://rubygems.org'
 
+gem 'celluloid',             github: 'celluloid/celluloid',             branch: '0.17.0-prerelease'
```
Comment (digitalextremist) [inline Gemfile]: @niamster this makes `celluloid-io` depend on the broken `0.17.0` release.

**Comment by digitalextremist** [comment]
Favoring #52

**Pull request closed without merging**